lib/utils.ts

```diff
@@ -16,3 +16,13 @@ export const envVarsPresent = () => {
 
   return !!gigsProject && !!gigsAPIKey
 }
+
+const privateEnvVars = {
+  APP_ENV: process.env.APP_ENV,
+  GIGS_PROJECT: process.env.GIGS_PROJECT,
+  GIGS_API_KEY: process.env.GIGS_API_KEY,
+}
+
+export function injectPrivateEnvVars() {
+  Object.assign(process.env, privateEnvVars)
+}
```

Comment:
I don't really understand what this does and why we need it 🙈

Reply:
I think I answered my question myself by missing to add this in the Tracking PR in the API Docs 😅

Reply:
Totally missed that question 😅 It's to ensure that relevant env variables are accessible when the project is built and running somewhere (or at least that's my understanding)